client/package.json

```diff
@@ -8,7 +8,7 @@
     "react-scripts": "3.1.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "BROWSER='Firefox' react-scripts start",
```

Comment:
When I have the React client running, it opens my browser automatically. I want it to open Firefox instead of Chrome.